classes/widgets/class-published-content-density.php

```diff
@@ -129,14 +146,15 @@ public function get_all_activities_density() {
 		// Get the all-time average.
 		static $density;
 		if ( null === $density ) {
-			$density = $this->count_density(
+			$activities = $this->filter_activities(
 				\progress_planner()->get_query()->query_activities(
```

Comment:
Why the `\` in front here? Isn't this in the same namespace?

Reply:
The `progress_planner()` function is in the main plugin file, so doesn't have a namespace.
It's just shorthand instead of typing `Base::get_instance()`, so we could even remove this function completely 🤔

Reply:
i'm fine with this too.